rs/registry/nns_data_provider/BUILD.bazel

```diff
@@ -9,6 +9,7 @@ DEPENDENCIES = [
     "//rs/crypto/tree_hash",
     "//rs/interfaces/registry",
     "//rs/nns/constants",
+    "//rs/registry/canister",
```

Comment:
This dependency is needed just for the availability of `SwapNodeInSubnetDirectlyPayload`. There are quite a couple of types that are added among the `mutations` part of the registry canister. Maybe we could add a `types` module in `registry/transport` and collect the hand-written types there? Some of those types are `*Directly` payloads, `ChangeSubnetMembership`... 

If you think this would be useful, I could do it for this pr as a start (just extracting the `SwapNodeInSubnetDirectlyPayload`) and then add a `TODO` for future to extract the others as well.

Reply:
Solved with what Max suggested.